src/css/styles.css

```diff
@@ -1,5 +1,104 @@
+/* * {
+  border: 1px solid greenyellow;
+} */
+
 body, html {
-  background-image: linear-gradient(to top, #ff9a9e 0%, #fecfef 99%, #fecfef 100%);
+  background-image: linear-gradient( #4D4C4C, #000);
   background-repeat: no-repeat;
-  height: 100%;
-}
\ No newline at end of file
+  background-attachment: fixed;
+  color: #fff;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.4em;
+  height: 100vh;
+  width: 100vw;
+  margin: 0;
+  overscroll-behavior: none;
+}
+
+main {
+  display: flex;
+  height: 100vh;
+  justify-content: space-between;
+}
+
+@media screen and (max-width: 851px) {
```

Comment:
Why 851px?  That doesn't seem like a standard breakpoint.

Reply:
It's not standard, but that is exactly where elements start to shift to compensate for the lack of space. We can discuss and make some changes if you want. I'm open to hearing your thoughts around it.